### ISSUE TITLE
Plone GitHub Team `Contributors` is gone

### ISSUE THREAD

**Issue by stevepiercy** [plone/ai-team#6]
Did someone delete the Plone GitHub Team Contributors? I verified it is gone.

https://community.plone.org/t/why-is-the-contributors-team-is-removed-from-the-github/17612

https://github.com/orgs/plone/teams

@fredvd @polyester @rudaporto @ericof @tkimnguyen @gforcada @davisagli @mauritsvanrees

**Comment by ericof** [comment]
We need to check the audit log.

**Comment by davisagli** [comment]
The audit log shows the @fredvd deleted it 2 days ago. I am also surprised by this.

<img width="631" alt="Screenshot 2023-06-29 at 3 32 21 PM" src="https://github.com/plone/ai-team/assets/49014/afdabf7f-a124-4d51-9dec-1bd641ae474b">

**Comment by gforcada** [comment]
I'm having breakfast with him, I will ask 😄

**Comment by fredvd** [comment]
Ouch !!!   That was not my intention at all.  I wanted to move 1 Plone org member up fron Contributor to Developer (a GSOC student)  so I added him to Developers Team and remove him from Contributors team. 

I have no  clue how I managed to remove the whole Contributors Team.  Will look into it ASAP.  

Pffff :-(

**Comment by fredvd** [comment]
I have to check the audit log because now that I think of what I did this week team wise I added the Classic UI team after that and and that was shorter ago.

**Comment by fredvd** [comment]
So the times all match up, as I chatted to @stevepiercy: just after having added justdaksh to the Developers time I deleted the Contributors team instead of removing him from the contributors team.  I'll check later with a test team what the UI for that is because I don't  have a clue how I managed to misinterpret the UI.

Restore procedure: I have just created an audit log export of all team.add_members activity and dates back long enough where I see myself creating the contributors team somewhere in March (and myself getting added to it) and all subsequent actions. by Andy to add new organisation members to the Contributors Team.

So I'll write a small script and use GitHub api to re-create the team and add all users again to the Contributors team.

The alternative or check would have been to  export a list of all Plone org members and subtract members from Developers Team. At the moment you are either a Contributor or a Developer.

Also functional AFAIK there are no permissions tied to the Contributors team.  Contributors fall back to the base permissions set on the Plone organisations for Members of the org on GitHub. We only added the Contributors team back in March to have a 'visual' collection of new members in the org.  Members of the Developers team get more permissions.

That's probably also the reason why nobody noticed so far and the first Contributor member reported this on the 29th when I accidentally deleted the team on the 27th of June.
